<?xml version="1.0"?>

<launch>
  <node name="detection_tracker" pkg="cob_people_object_detection_tensorflow" type="cob_people_object_detection_tensorflow.py" output="screen">
    <rosparam command="load" file="$(find cob_people_object_detection_tensorflow)/launch/cob_people_object_detection_tensorflow_params.yaml"/>
  </node>
</launch>
